From 6c3b4c491ee013b225e664441c96e3489d5d90c1 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 10 May 2017 17:30:10 -0700
Subject: [PATCH] TOOLS-1775 correct mg/tools/launch-build output when using a
 TRY_BRANCH build via -t Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
 Approved by: Cody Peter Mello <cody.mello@joyent.com>

---
 tools/launch-build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/launch-build b/tools/launch-build
index a46cf29..f4e3e55 100755
--- a/tools/launch-build
+++ b/tools/launch-build
@@ -61,7 +61,7 @@ if [[ -n $BRANCH ]]; then
 	printf "Building %s with BRANCH=%s\n" "$PROJECT" "$BRANCH"
 	BUILD_PARAM=`printf '{"name":"BRANCH", "value": "%s"}' $BRANCH`
 elif [[ -n $TRY_BRANCH ]]; then
-	printf "Building %s with TRY_BRANCH=%s\n" "$PROJECT" "$BRANCH"
+	printf "Building %s with TRY_BRANCH=%s\n" "$PROJECT" "$TRY_BRANCH"
 	BUILD_PARAM=`printf '{"name":"TRY_BRANCH", "value": "%s"}' $TRY_BRANCH`
 fi
 
-- 
2.21.0

